<?xml version="1.0" encoding="UTF-8"?><testrun name="PessoaServiceTest" project="curso-jpa" tests="4" started="0" failures="0" errors="1" ignored="0">
  <testsuite name="br.java.curso.pessoa.PessoaServiceTest" incomplete="true">
    <error>java.lang.IllegalArgumentException: Unknown entity: org.hibernate.jpa.internal.EntityManagerImpl
	at org.hibernate.jpa.spi.AbstractEntityManagerImpl.remove(AbstractEntityManagerImpl.java:1182)
	at br.java.curso.pessoa.PessoaServiceTest.clear(PessoaServiceTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.hibernate.MappingException: Unknown entity: org.hibernate.jpa.internal.EntityManagerImpl
	at org.hibernate.internal.SessionFactoryImpl.getEntityPersister(SessionFactoryImpl.java:776)
	at org.hibernate.internal.SessionImpl.getEntityPersister(SessionImpl.java:1533)
	at org.hibernate.event.internal.DefaultDeleteEventListener.onDelete(DefaultDeleteEventListener.java:82)
	at org.hibernate.event.internal.DefaultDeleteEventListener.onDelete(DefaultDeleteEventListener.java:56)
	at org.hibernate.internal.SessionImpl.fireDelete(SessionImpl.java:924)
	at org.hibernate.internal.SessionImpl.delete(SessionImpl.java:863)
	at org.hibernate.jpa.spi.AbstractEntityManagerImpl.remove(AbstractEntityManagerImpl.java:1179)
	... 17 more

</error>
    <testcase name="salvarBancoTeste" classname="br.java.curso.pessoa.PessoaServiceTest" incomplete="true"/>
    <testcase name="buscarTodosTeste" classname="br.java.curso.pessoa.PessoaServiceTest" incomplete="true"/>
    <testcase name="compararNomes" classname="br.java.curso.pessoa.PessoaServiceTest" incomplete="true"/>
    <testcase name="buscarPessoaPorIdTeste" classname="br.java.curso.pessoa.PessoaServiceTest" incomplete="true"/>
  </testsuite>
</testrun>
